src/features/shared-link-settings-modal/SharedLinkSettingsModal.js

```diff
@@ -220,9 +221,19 @@ class SharedLinkSettingsModal extends Component {
     };
 
     renderVanityNameSection() {
-        const { canChangeVanityName, serverURL, vanityNameInputProps, warnOnPublic = false } = this.props;
+        const {
+            canChangeVanityName,
+            hideVanityNameSection,
+            serverURL,
+            vanityNameInputProps,
+            warnOnPublic = false,
+        } = this.props;
         const { vanityNameError, isVanityEnabled } = this.state;
 
+        if (hideVanityNameSection) {
```

Comment:
Nitpick: Could we do this instead? ```{!hideVanityNameSection ? <VanityName... /> : null}```

Reply:
That's exactly how I first wrote it then I changed it to this to be more consistent with other box react `render()` methods.